redis/asyncio/connection.py

```diff
@@ -827,7 +831,9 @@ async def can_read_destructive(self):
     async def read_response(
         self,
         disable_decoding: bool = False,
+        *,
```

Comment:
Ah, it was an afterghought.  I was the one who added the timeout argument originally and in retrospect it probably should have been a keyword-only argument.  Since it was _my_ addition, I thought changing it retro-actively to a kw-only would be ok (there was only a two-month inverval between those two changes).  "timeout" is traditionally a kw-only arg these days.
I guess one should be careful with these things, so I´m moving the asterisk :)